docs/guide/ja/basics.md

```
@@ -188,6 +188,11 @@ Claude でも Codex でも同じです。**Agent Picker** でどちらかを選
   [設定で置き換え可](config.html#header)）・**Activity timeline**（時計）＝ツール呼び出し履歴。
   接続状態は接続中・失敗時だけここに出ます。
 
+> **ファイルツリーの行を右クリック**すると、そのファイルのパスをターミナルのカーソル位置に
+> 挿入できます —— *Insert relative path* / *Insert absolute path*。行の上で `Shift+F10` でも
+> 同じものが開きます。ディレクトリでも使えます。相対パスの項目は、ツリーとターミナルが
+> 同じディレクトリにあるときだけ出ます（そのときだけ両端で同じ意味になるため）。
+
```

markdownlint-cli2 (0.23.2)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)